dandiapi/api/tests/test_version.py

```diff
@@ -654,8 +654,8 @@ def test_version_rest_publish_zarr(
         f'/api/dandisets/{draft_version.dandiset.identifier}'
         f'/versions/{draft_version.version}/publish/'
     )
-    assert resp.status_code == 400
-    assert resp.json() == ['Cannot publish dandisets which contain zarrs']
+    assert resp.status_code == 403
```

Comment:
Just a note to back out this one change we discussed.

Reply:
:+1: